tc/sgx/trusted_worker_manager/enclave/enclave_data.cpp

```diff
@@ -99,6 +110,10 @@ void EnclaveData::DeserializeSealedData(const std::string& inSerializedEnclaveDa
     svalue.assign(pvalue);
     public_signing_key_.Deserialize(svalue);
 
+    // Clear local variables storing secrets before re-assigning new value
+    svalue.clear();
+    std::string(pvalue).clear();
```

Comment:
Directly clear the contents of pvalue ( memset_s). std:string() constructor from char* does a copy.

Reply:
Ok